chicken-sloths/bangazon-cli-sprint2
# Select Active Customer Prompts

## I know this has merge conflicts! I'll fix them all right before I merge

# Description
This covers the prompts for selecting and setting an active customer

## Related Ticket(s)
Related to Issue #2, does everything EXCEPT get all the customers

## Problem to Solve
Creates menu option and sub-prompt for setting active customer

## Steps to Test Solution

1. Run ``npm run start`` to see the main menu
1. Select option ``2``
1. Pretend like you see a long list of possible customers
1. Type a number (this represents the imaginary customer id)

## Expected Behavior
1. You should see that same number console.logged back out at you.

## Notes
I made the function that sets the active customer private to ``controllers/activeCustC.js``. (Previously it was exported, and I guess we could call it in ui.js when the promise resolves? But I figure we might as well just call it in that module for simplicity's sake.)



## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods
